.github/workflows/ci.yml

```diff
@@ -125,10 +125,45 @@ jobs:
       - name: Run Front-end Tests
         run: |
           blt tests:frontend:run --define drush.alias='${drush.aliases.ci}' --define tests.run-server=false --environment=ci --ansi --verbose --no-interaction
+      - name: Install cypress
+        run: |
+              blt tests:cypress:npm-install --define drush.alias='${drush.aliases.ci}' --environment=ci --ansi --verbose --no-interaction
+      - name: Start test server
+        run: |
+              ## Running drush runserver is equivelent to blt tests:server:start, and is what happens under the hood, except this allows us to bind to all ips on the box.
+              ## blt was not letting us bind to those IPs we wanted.
+              drush runserver --quiet 0.0.0.0:8888 &
+      - name: Cypress pre-tests
+        timeout-minutes: 5
+        run: |
+              blt tests:cypress:pretest --define drush.alias='${drush.aliases.ci}' --define tests.run-server=false --environment=ci --ansi --verbose --no-interaction
+      - name: Upload pretest screenshots
+        if: failure()
+        uses: actions/upload-artifact@v4
+        with:
+          name: cypress-pretest-screenshots
+          path: ./tests/cypress/cypress/screenshots
       - name: Behavioral Tests
         timeout-minutes: 120
         run: |
-          echo "TODO: Add Cypress tests here"
+              blt tests:cypress:runimpl --define drush.alias='${drush.aliases.ci}' --define tests.run-server=false --environment=ci --ansi --verbose --no-interaction
```

Comment:
This should be run to follow the other test naming conventions.

Reply:
Done.